middle.mkd

```diff
@@ -189,6 +189,30 @@ Implementations SHOULD NOT extend positions beyond 3 elements. Parsers
 MAY ignore additional elements. Interpretation and meaning of additional
 elements is beyond the scope of this specification.
 
+A line between two positions is a straight Cartesian line, the shortest
```

Comment:
Should this be parsed as equivalent to

`A line between two positions MUST be interpreted as a straight Cartesian line, the shortest`

or 

`A line between two positions SHOULD be interpreted as a straight Cartesian line, the shortest`

or is it entirely normative?